### PR TITLE
* Change to use image from gcr

### PR DESCRIPTION
**Pull request by pugangxa**
**What this PR does / why we need it**:
Change to use image from gcr
Fix the hostname to predict

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/544)

### CODE REVIEW

File: docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb [hunk 1]
```diff
@@ -158,23 +162,21 @@
...
-      "[[-1.6099601984024048, -2.6461076736450195, 0.32844462990760803, 2.4825074672698975, 0.43524616956710815, 2.3108043670654297, 1.00056791305542, -0.4232763648033142, -0.5100948214530945, -1.7978394031524658]]\n",
-      "3\n"
-     ]
+     "text": "http://9.21.53.162/v1/models/cifar10:predict {'Host': 'cifar10.kubeflow.example.com'}\n[[-1.6099603176116943, -2.6461076736450195, 0.3284446597099304, 2.4825077056884766, 0.43524613976478577, 2.3108043670654297, 1.0005676746368408, -0.4232763648033142, -0.5100947022438049, -1.7978397607803345]]\n3\n"
     }
    ],
    "source": [
     "url = \"http://\" + cluster_ip + \"/v1/models/cifar10:predict\"\n",
-    "headers = { 'Host': 'cifar10-predict.kubeflow.example.com' }\n",
+    "headers = { 'Host': 'cifar10.kubeflow.example.com' }\n",
```
Comment (yuzisun) [inline docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb]: can you help fix the namespace to not deploy to Kubeflow which is system namespace? thanks.

File: docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb [hunk 2]
```diff
@@ -244,4 +237,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
+}
```
Comment (yuzisun) [inline docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb]: nit: newline

**Comment by ellis-bigelow** [comment]
Nice! Will let yuzi approve /lgtm

File: docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb (hunk 1 above)
Comment (pugangxa) [reply docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb]: Fixed

File: docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb (hunk 2 above)
Comment (pugangxa) [reply docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb]: Updated

**Pull request merged**